graphql_api/actions/flags.py

```diff
@@ -48,25 +48,11 @@ def flag_measurements(
     after: datetime,
     before: datetime,
 ) -> Mapping[int, Iterable[dict]]:
```

Comment:
Move this to a shared generic function to be used by both flag and components measurements